docs/src/manifolds/hyperbolic.md

```diff
@@ -28,10 +28,94 @@ Pages = ["manifolds/HyperbolicPoincareBall.jl"]
 Order = [:type, :function]
 ```
 
+### Plotting
```

Comment:
Or at least add some links at the top of this page that will make it clear that there are visualizations below.

Reply:
The idea for both Plottings is as follows: They are sub-sections of the representations, hence third order, and I wanted to first introduce the structs/types so that's why they are after the autodocs area.

Reply:
Can you make the request with the links a little more specific? I am not sure what to do there.

Reply:
Maybe something similar to the list at the top of this page: https://juliadocs.github.io/Documenter.jl/stable/man/syntax/ that has links to these three models of `Hyperbolic` and notes that there are visualizations available?

Reply:
I will check that tomorrow.